editor.planx.uk/src/@planx/components/Section/Section.stories.tsx

```diff
@@ -14,13 +14,73 @@ type Story = StoryObj<typeof meta>;
 
 export default meta;
 
-// TODO needs internal state
+const sectionNodes: { [key: string]: SectionNode } = {
+  firstSection: {
+    data: {
+      title: "First section",
+    },
+    type: TYPES.Section,
+  },
+  secondSection: {
+    data: {
+      title: "Second section",
+    },
+    type: TYPES.Section,
+  },
+  thirdSection: {
+    data: {
+      title: "Third section",
+    },
+    type: TYPES.Section,
+  },
+};
+
+const defaultProps: ComponentProps<typeof Public> = {
+  currentSectionIndex: 0,
+  flowName: "Find out if you need planning permission",
+  sectionNodes,
+  currentCard: {
+    id: "abc123",
+  },
+  flow: {
+    _root: {
+      edges: ["firstSection", "secondSection", "thirdSection"],
+    },
+    ...sectionNodes,
+  },
+  changeAnswer: () => console.log("changeAnswer called"),
+  breadcrumbs: {},
+  title: "The property",
+  sectionCount: 3,
+};
+
 export const Basic = {
+  args: defaultProps,
+} satisfies Story;
+
+export const FirstSectionCompleted = {
   args: {
-    title: "The property",
+    ...defaultProps,
+    currentSectionIndex: 1,
+    breadcrumbs: {
+      firstSection: {
+        auto: false,
+      },
+    },
   },
 } satisfies Story;
 
-export const WithEditor = () => {
-  return <Wrapper Editor={Editor} Public={Public} />;
-};
```

Comment:
One downside - I can't get this approach to work with the `Wrapper` component. Not a huge issue for this single component which has a simple Editor representation but it probably shows that this isn't an approach we want to repeat too much.